configurations/coding_plans.py

```diff
@@ -139,7 +146,8 @@ def get_rqa_coding_plans(pipeline_name):
                        include_in_theme_distribution=Codes.TRUE
                    ),
                ],
-               ws_code=CodeSchemes.KAKUMA_WS_CORRECT_DATASET.get_code_with_match_value("kakuma location"),
+               ws_code=CodeSchemes.KAKUMA_WS_CORRECT_DATASET.get_code_with_match_value(
```

Comment:
This makes it even harder to read the code here. In PEP 8: We only break a line if the characters are too long i.e > 120